src/main/java/fi/tkgwf/ruuvi/utils/InfluxDBConverter.java

```diff
@@ -59,6 +59,9 @@ public static Point toInflux(EnhancedRuuviMeasurement measurement, Predicate<Str
         if (measurement.getDataFormat() != null) {
             p.tag("dataFormat", String.valueOf(measurement.getDataFormat()));
         }
+        if (measurement.getReceiver() != null) {
```

Comment:
The receiver is actually never null (at least the way it is implemented now), perhaps use `if (StringUtils.isNotBlank(measurement.getReceiver())) {` instead?

Reply:
Excellent point :). I thought of using null in one point, but then decided just to use blank.